client/src/components/Admin/Navbar.js

```diff
@@ -11,14 +11,7 @@ const Navbar = () => {
         >
```

Comment:
you should have used a different branch for raising a new PR. Now see, both your PRs has the same set of changes. Always create new branch for every PR

Reply:
sure @iMagesh I actually forgot that and commited the code, from next time I will create new branch for every PR.